services/app-api/src/handlers/reports.ts

```diff
@@ -91,10 +109,26 @@ export const main: APIGatewayProxyHandler = async () => {
             console.error('Error decoding revision')
             throw new Error(`Error generating reports array`)
         } else {
-            bucket.push(revision)
+            // add the package name to the revision
```

Comment:
This all looks right. 

I haven’t looked through this handler before, how does this get invoked? I don’t see anything limiting this to ADMIN users which I thought was a requirement for this.

Reply:
It's restricted to CMS users at the route level.

Reply:
Can you show me? I see us restricting to logged in users in serverless.yml but I don’t see any restriction based on user role

Reply:
Added a bit to restrict the endpoint itself to cms and admin users 🙂

Reply:
Nice, yay!